tools/pipelines/templates/include-telemetry-setup.yml

```diff
@@ -83,15 +87,17 @@ steps:
   inputs:
     workingFile: ${{ parameters.pathToTelemetryGenerator }}/.npmrc
 
-# Install logger package
-- task: Npm@1
-  displayName: 'npm install aria logger'
+- task: Bash@3
+  displayName: 'pnpm install aria logger'
   retryCountOnTaskFailure: 1
   inputs:
-    workingDir: ${{ parameters.pathToTelemetryGenerator }}
-    command: 'custom'
-    customCommand: 'install @ff-internal/aria-logger'
-    customRegistry: 'useNpmrc'
+    targetType: 'inline'
+    workingDirectory: ${{ parameters.pathToTelemetryGenerator }}
+    script: |
+      set -eu -o pipefail
+      # The telemetry-generator package is the root of its workspace so we do want to install this for the workspace
+      # root with '-w'. Without it, the install will complain and fail.
+      pnpm install -w @ff-internal/aria-logger;
```

Comment:
We do a specific install of just `@ff-internal/aria-logger` and then later a full install. Why?

Reply:
Because aria-logger is not part of the dependencies of the package; as an internal-only part of our infra, we're basically adding it to the package at this point during the pipeline run, and then the full install is to bring in all the deps that were already in package.json.

Reply:
The pnpm command to add a dependency is `add`. I have only seen `install` used for installing existing dependencies. If this is an extra installation and not part of the regular deps, I would recommend it happen after regular install. Actually, I am surprised the regular install after this doesn't wipe this out.

Reply:
And since there is not version specified here, I am assuming that we are always happy with it picking the latest it sees.
This feels like a toolchain/SBOM hole.

Reply:
Yeah, it currently is. At some point it was a deliberate decision to let the version float so that improvements in aria-logger didn't require a separate change here to start having effect, but last time this topic came up in some private chat, I think we agreed pinning it would make sense, especially to prevent breaking older branches like LTS. That wouldn't prevent newer versions of transitive dependencies being brought in though, unless we first pin all those in aria-logger, which might be the right thing to do.

Reply:
If we are going to pin, shouldn't we just make it part of regular dependencies and the lockfile?